features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/index.jsx

```diff
@@ -47,14 +46,22 @@ import Utils from 'AppData/Utils';
 import ResourceNotFound from 'AppComponents/Base/Errors/ResourceNotFound';
 import CustomIcon from 'AppComponents/Shared/CustomIcon';
 import LeftMenuItem from 'AppComponents/Shared/LeftMenuItem';
+
 import API from 'AppData/api';
 import APIProduct from 'AppData/APIProduct';
+
```

Comment:
Shall we remove this unnecessary white spaces

Reply:
Fixed